### PR TITLE
[infocyte-604] Infocyte Pack

### PR DESCRIPTION
**Pull request by singlethreaded**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Initial PR for Infocyte Integration review. All primary commands validated in container manually and in Demisto via CLI.

## Known Issues
- Invoke-Pester does not work from the sdk as it fails to find the Infocyte module in the container. Running Invoke-Pester from inside the container manually works.
- Validating Unit Tests does not work yet, not enough documentation on the format of the $Demisto.Results object following a command run
- GetLastRun format is unknown so unable to get alerts from a last run
- Test Playbook missing async task status loop

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by singlethreaded** [comment]
https://github.com/demisto/content/pull/7480#issuecomment-641715856
API Key is no longer valid so not a timely issue: I'll remove the reference as it was for some manual testing.

File: Tests/conf.json [hunk 1]
```diff
@@ -2873,6 +2878,7 @@
         "Endpoint Enrichment - Generic v2 - Test": "DEPRECATED"
     },
     "skipped_integrations": {
+        "Infocyte": "In development",
```
Comment (Itay4) [inline Tests/conf.json:2881]: no need to skip it, please remove

File: Packs/Infocyte/Integrations/Infocyte/README.md [hunk 2]
```diff
@@ -0,0 +1,484 @@
+Infocyte can pivot off incidents to automate triage, validate events with forensic data and enabling dynamic response actions against any or all host using both agentless or agented endpoint access.
+This integration was integrated and tested with version xx of Infocyte
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/README.md:2]: please add the product version here

File: Packs/Infocyte/Integrations/Infocyte/README.md [hunk 3]
```diff
@@ -0,0 +1,484 @@
...
+| Infocyte.Scan.compromised | boolean | Flagged if system has a malicious item found | 
+| Infocyte.Scan.alerts | array | List of alerts associated with scan of host | 
+| Infocyte.Scan.alertCount | number | number of alerts associated with scan of host | 
+| Infocyte.Scan.hostname | string | Hostname of target host | 
+| Infocyte.Scan.ip | string | Ip of target host | 
+| Infocyte.Scan.compromisedObjects | number | Bad files, artifacts, and events found | 
+| Infocyte.Scan.objectCount | number | Total number of files, artifacts, and events inspected or retrieved | 
+
+
+#### Command Example
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/README.md:390]: a few sections in the README as missing, such as this one
please fill them up

File: Packs/Infocyte/Integrations/Infocyte/Infocyte.yml [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+  id: Infocyte
+  version: -1
+name: Infocyte
+display: Infocyte
+category: Endpoint
+description: Infocyte can pivot off incidents to automate triage, validate events
+  with forensic data and enabling dynamic response actions against any or all host
+  using both agentless or agented endpoint access.
+configuration:
+- display: Instance Name
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/Infocyte.yml:11]:
```suggestion
- display: Instance Name (e.g., https://<cname>.infocyte.com)
```

File: Packs/Infocyte/Integrations/Infocyte/Infocyte.yml [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+      description: Hostname or ip of target provided
+      type: string
+    - contextPath: Infocyte.Task.extensionName
+      description: Name of extension ran
+      type: string
+  - name: infocyte-get-taskstatus
+    description: Gets status of an Infocyte task (scan, response action, etc.)
+    arguments:
+    - name: userTaskId
+      description: Task id used with infocyte-get-taskstatus
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/Infocyte.yml:191]: please mention here how this ID can be retrieved

File: Packs/Infocyte/Integrations/Infocyte/Infocyte.yml [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+        failed)
+      type: string
+    - contextPath: Infocyte.Task.timeElapsed
+      description: Seconds since task created
+      type: number
+  - name: infocyte-get-scanresult
+    description: Retrieve metadata and results for a scan against multiple hosts
+    arguments:
+    - name: scanId
+      description: Infocyte id used to look up the data associated with a specific
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/Infocyte.yml:221]: please mention here how this ID can be retrieved

File: Packs/Infocyte/Integrations/Infocyte/Infocyte.yml [hunk 7]
```diff
@@ -0,0 +1,406 @@
...
+      description: Bad files, artifacts, and events found
+      type: number
+    - contextPath: Infocyte.Scan.objectCount
+      description: Total files, artifacts, and events collected
+      type: number
+  - name: infocyte-get-hostscanresult
+    description: Retrieve results for a scan on a target host
+    arguments:
+    - name: scanId
+      description: Infocyte id used to look up the data associated with a specific
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/Infocyte.yml:248]: please mention here how this ID can be retrieved

File: Packs/Infocyte/Integrations/Infocyte/Infocyte.yml [hunk 8]
```diff
@@ -0,0 +1,406 @@
...
+    - contextPath: Infocyte.Scan.compromisedObjects
+      description: Bad files, artifacts, and events found
+      type: number
+    - contextPath: Infocyte.Scan.objectCount
+      description: Total number of files, artifacts, and events inspected or retrieved
+      type: number
+  - name: infocyte-get-responseresult
+    description: "Gets the results of a response action."
+    arguments:
+    - name: scanId
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/Infocyte.yml:288]: please mention here how this ID can be retrieved

File: Packs/Infocyte/Integrations/Infocyte/Infocyte.yml [hunk 9]
```diff
@@ -0,0 +1,406 @@
...
+    - contextPath: Infocyte.Response.extensionId
+      description: Id of Infocyte extension being run
+      type: string
+    - contextPath: Infocyte.Response.extensionName
+      description: Name of Infocyte extension being run
+      type: string
+  - name: infocyte-get-alerts
+    description: Retrieve alerts since last alert pulled
+    arguments:
+    - name: alertId
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/Infocyte.yml:337]: please mention here how this ID can be retrieved

File: Packs/Infocyte/Integrations/Infocyte/Pipfile [hunk 10]
```diff
@@ -0,0 +1,18 @@
+[[source]]
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/Pipfile:1]: No need for the Pipfile

File: Packs/Infocyte/Integrations/Infocyte/Pipfile.lock [hunk 11]
```diff
@@ -0,0 +1,369 @@
+{
```
Comment (Itay4) [inline Packs/Infocyte/Integrations/Infocyte/Pipfile.lock:1]: No need for the Pipfile.lock

**Comment by Itay4** [comment]
please make sure the integration logo is according to [the standards](https://xsoar.pan.dev/docs/integrations/integration-logo)

**Comment by singlethreaded** [comment]
ready for another look. Won't be able to pass CircleCI tests till your sdk Pester PR is pulled in.

**Comment by kirbles19** [comment]
@Itay4 Done.

**Comment by glicht** [comment]
@singlethreaded I think you added the following file by mistake: `packages-microsoft-prod.deb`

**Comment by singlethreaded** [comment]
> Please remove: `packages-microsoft-prod.deb`

I don't see this file anywhere

**Comment by Itay4** [comment]
> > Please remove: `packages-microsoft-prod.deb`
> 
> I don't see this file anywhere

@singlethreaded https://github.com/demisto/content/blob/00abe4f4f908557ed74e60492074fc925a43f1e2/packages-microsoft-prod.deb

**Comment by singlethreaded** [comment]
Good for merge. Playbook runs. All functions check.

**Pull request merged**